dietza/Overlook
# Set-up & Login Display

### What is the change?
Includes beginning structure and styling for login page in index.html and base.scss.
Builds initial structure for User class,
Early functionality for verifying login inputs and submitting fetch request to Users API

### What does it fix?
Instantiates users with valid login credentials,
uses fetched data

### Is this a fix or a feature? 
feature

### Where should the reviewer start?
line 4 index.html,
line 4 base.scss,
line 11 index.js

### How should this be tested?
Run npm start in your terminal and make sure the login page loads on the DOM (localhost:8080) and console is clear of bugs.
If a user does not input a valid username or password, they should be prompted to correct the input.
